taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=100, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE, related_name="manufacturers")
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="drivers")
```

Comment:
the related_name attribute in Django models is used to create a reverse relation from the related model back to the model defining the ForeignKey or ManyToManyField.

 So, it would be clearer to use a name like "cars" for the related_name. That way, if you have an instance of Manufacturer, you can get all the cars it manufactures using manufacturer_instance.cars.all().

the same for drivers